lib/SILGen/SILGenApply.cpp

```diff
@@ -5539,6 +5539,29 @@ RValue SILGenFunction::emitApplyAllocatingInitializer(SILLocation loc,
   return result;
 }
 
+RValue SILGenFunction::emitApplyPropertyDelegateAllocator(SILLocation loc,
+                                                          SubstitutionMap subs,
+                                                          SILDeclRef ctorRef,
+                                                          Type delegateTy,
+                                                    CanAnyFunctionType funcTy) {
+  Callee callee = Callee::forDirect(*this, ctorRef, subs, loc);
```

Comment:
Is this always a direct call?

Reply:
As far as I understand, yes.

Reply:
What if the initializer is `@objc dynamic`? Do we need to diagnose that case?